install-dev/models/install.php

```diff
@@ -910,15 +910,16 @@ public function installFixtures($entity = null, array $data = array())
         return true;
     }
 
-    public function installTheme()
+    public function installTheme($themeName = null)
     {
+        $themeName = $themeName ?: _THEME_NAME_;
         $builder = new ThemeManagerBuilder(
             Context::getContext(),
             Db::getInstance()
         );
 
         $theme_manager = $builder->build();
 
-        return $theme_manager->install(_THEME_NAME_) && $theme_manager->enable(_THEME_NAME_);
+        return $theme_manager->install($themeName) && $theme_manager->enable($themeName);
```

Comment:
```php
return $theme_manager->enable($themeName);
```
would be enough, right ? The install() is needed when we have a zip to extract. (ping @julienbourdeau)